open-feature/flagd
# refactor: isolate flag state management to store

## This PR


- Isolates flag state management to store package

### Related Issues


Fixes #371 

### Notes


### Follow-up Tasks



### How to test

I've ran [flagd's integration tests](https://github.com/open-feature/flagd/pull/312) against this build to ensure behaviour is as before.
